### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by victory-nwawunze**
I have finish creating the navbar component

### CODE REVIEW

File: common/Button/index.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const Button: React.FC<Props> = (props) => {
 
   const isColors = textColor || bg;
   const colors = {
-    color: textColor || '#fff',
+    // color: textColor || '#fff',
```
Comment (onyegood) [inline common/Button/index.tsx:17]: Why did you remove this line? The only thing you can adjust here is to make `#fff` a reusable variable.

File: common/Navbar/Navbar.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.white {
```
Comment (onyegood) [inline common/Navbar/Navbar.scss:1]: This class name should be called `bg-white`

File: common/Navbar/Navbar.scss
Comment (onyegood) [inline common/Navbar/Navbar.scss:1]: All these CSS classes are generic and should be moved to global.css for reusable sake.

File: common/Navbar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  };
+
+  return (
+    <nav className='white w-full  py-2 shadow lg:px-[97px]'>
+      <div className='container mx-auto'>
+        <div className='flex items-center justify-between '>
+          <Image alt='' src='/assets/EFICKOKIDS-Logo.svg' width={150} height={150} />
+
+          <div className='hidden space-x-10 md:flex'>
+            <a href='#' className='text-gray font-[Inter] text-[16px] font-[500px]'>
```
Comment (onyegood) [inline common/Navbar/index.tsx:22]: This `className='text-gray font-[Inter] text-[16px] font-[500px]'` is repetition make a custom class for it. And once you have more than 3 classes create a custom class for them.

File: common/Button/index.tsx (hunk 1 above)
Comment (victory-nwawunze) [reply common/Button/index.tsx:17]: Solved.

File: common/Navbar/Navbar.scss (hunk 2 above)
Comment (victory-nwawunze) [reply common/Navbar/Navbar.scss:1]: Change

File: common/Navbar/index.tsx (hunk 3 above)
Comment (ayotidapo) [reply common/Navbar/index.tsx:22]: All these classes you have here "font-[Inter] text-[16px] font-[500px]'"  already exist

@victory-nwawunze

Comment (victory-nwawunze) [reply common/Navbar/index.tsx:22]: All CSS styles are now reuseable.

Comment (victory-nwawunze) [reply common/Navbar/index.tsx]: @ayotidapo should i change the font family

**Review by onyegood — APPROVED** [review]

File: common/Navbar/Navbar.scss (hunk 2 above)
Comment (ayotidapo) [reply common/Navbar/Navbar.scss]: why bg-white there is text-white alreardy @onyegood @victory-nwawunze

**Review by ayotidapo — APPROVED** [review]
